### PR TITLE
asynchronous processing the make_osd_fs

### PR DESCRIPTION
**Pull request by liangzhaowang**
Signed-off-by: Zhaowangx.Liang <zhaowangx.liang@intel.com>

### CODE REVIEW

File: workflow/workflow.py [hunk 1]
```diff
@@ -46,21 +47,20 @@ def run(self):
...
         osds = self.cluster["osds"]
         pwd = os.path.abspath(os.path.join('..'))
+        run_deploy_args = ['redeploy']
         if len(self.cluster["rgw"]) and self.cluster["rgw_enable"]=="true":
-            with_rgw = True
-        else:
-            with_rgw = False
+            run_deploy_args.append('--with_rgw')
+        if not (self.cluster.has_key("parallel_deploy") and self.cluster["parallel_deploy"]=="false"):
+            run_deploy_args.append('--parallel_deploy')
```
Comment (xuechendi) [inline workflow/workflow.py]: why append '--parallel_deploy' when it set false?

**Comment by xuechendi** [comment]
@Shaoshian , please help to test this PR on your cluster

File: workflow/workflow.py (hunk 1 above)
Comment (liangzhaowang) [inline workflow/workflow.py]: there is a 'not' before the conditions

Comment (xuechendi) [inline workflow/workflow.py]: @liangzhaowang , oh, I see, but this is not desired way to do so, cetune will set default for all parameter, please ask shian for the way to do so

**Comment by Shaoshian** [comment]
@xuechendi   目前还存在一些小问题

**Pull request closed without merging**